client/stream_alerting_rule.go

```diff
@@ -58,7 +58,7 @@ type StreamAlertingRuleResponseRelationships struct {
 func (c *Client) CreateAlertingRule(
 	ctx context.Context,
 	projectName string,
-	updateInterval int,
+	updateIntervalMs int,
```

Comment:
this is unrelated, it was just bugging me that there were no units on this thing